### PR TITLE
[SA-454][SA-441]

### PR DESCRIPTION
**Pull request by aditkpt**

### CODE REVIEW

File: src/components/Atoms/Iconography/toolbar-bold.tsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+const Bold = () => (
+    <svg
+        xmlns="http://www.w3.org/2000/svg"
+        height="24px"
+        viewBox="0 0 24 24"
+        width="24px"
+        fill="#007ACC"
```
Comment (rioarray) [inline src/components/Atoms/Iconography/toolbar-bold.tsx:9]: please use fluids/colors for defaut colors @aditkpt

File: src/components/Atoms/Iconography/toolbar-bullet-list.tsx [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+const BulletList = () => (
+    <svg
+        xmlns="http://www.w3.org/2000/svg"
+        height="24px"
+        viewBox="0 0 24 24"
+        width="24px"
+        fill="#007ACC"
```
Comment (rioarray) [inline src/components/Atoms/Iconography/toolbar-bullet-list.tsx:9]: please use fluids/colors for defaut colors @aditkpt

File: src/components/Atoms/Iconography/toolbar-italic.tsx [hunk 3]
```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+const Italic = () => (
+    <svg
+        xmlns="http://www.w3.org/2000/svg"
+        height="24px"
+        viewBox="0 0 24 24"
+        width="24px"
+        fill="#007ACC"
```
Comment (rioarray) [inline src/components/Atoms/Iconography/toolbar-italic.tsx:9]: please use fluids/colors for defaut colors @aditkpt

File: src/components/Atoms/Iconography/toolbar-number-list.tsx [hunk 4]
```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+const NumberList = () => (
+    <svg
+        xmlns="http://www.w3.org/2000/svg"
+        height="24px"
+        viewBox="0 0 24 24"
+        width="24px"
+        fill="#007ACC"
```
Comment (rioarray) [inline src/components/Atoms/Iconography/toolbar-number-list.tsx:9]: please use fluids/colors for defaut colors @aditkpt

File: src/components/Atoms/Iconography/toolbar-underline.tsx [hunk 5]
```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+const Underline = () => (
+    <svg
+        xmlns="http://www.w3.org/2000/svg"
+        height="24px"
+        viewBox="0 0 24 24"
+        width="24px"
+        fill="#007ACC"
```
Comment (rioarray) [inline src/components/Atoms/Iconography/toolbar-underline.tsx:9]: please use fluids/colors for defaut colors @aditkpt

File: src/components/Atoms/Pills/Pills.images.tsx [hunk 6]
```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+export interface CrossIconProp {
+    color?: string;
+}
+
+export const Cross = ({ color = '#797F86' }: CrossIconProp) => (
```
Comment (rioarray) [inline src/components/Atoms/Pills/Pills.images.tsx:7]: please use fluids/colors for defaut colors @aditkpt

**Review by rioarray — CHANGES_REQUESTED** [review]

**Review by maulidiyah-nur — APPROVED** [review]

**Review by ijalandhika — APPROVED** [review]

**Pull request merged**